src/components/sw360/AdvancedSearch/AdvancedSearch.tsx

```diff
@@ -64,7 +64,9 @@ function AdvancedSearch({ title = 'Advanced Search', fields }: Props): JSX.Eleme
             }
         })
 
-        const encodedUrl = encodeURI(searchUrl.toString())
+        const encodedUrl = encodeURI(searchUrl.toString().includes('%40') ?
```

Comment:
Can we replace this encode snippet as below?
```suggestion
       const encodedUrl = encodeURI(searchUrl.toString().replace(/%40/g, '@'));
```

Reply:
@hoangnt2 Fixed ! 👍🏻